content/management/configuration-api/index.mdx

```diff
@@ -2654,7 +2654,7 @@ Enables the webhooks [registered](#register-webhook) for a given **Client Id** (
 
 | Parameter         | Required | Data type | Notes                                                                                                                  |
 | ----------------- | -------- | --------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `owner_client_id` | Yes      | `string`  | Required when authorizing via [PATs](/authorization/authorizing-api-calls/#personal-access-tokens); ignored otherwise. |
+| `owner_client_id` | Yes      | `string`  | Required when authorizing via [PATs](/authorization/authorizing-api-calls/#personal-access-tokens); rejected otherwise. |
```

Comment:
```suggestion
| `owner_client_id` | Yes      | `string`  | Required only when authorizing via [PATs](/authorization/authorizing-api-calls/#personal-access-tokens). The request will be rejected if you provide `owner_client_id` when authorizing with a Bearer Token. |
```

Reply:
"rejected otherwise" was grammatically referring to `owner_client_id`, but it's the **request** that will be rejected, not the param.

I also extended the description because I thought this part with "Otherwise" was not specific enough.